pkg/webhook/webhook.go

```diff
@@ -68,54 +70,57 @@ func (e *EnvoyFilterWebhook) createAdmissionResponse(
 		return admission.Errored(http.StatusInternalServerError, err)
 	}
 
-	filters := []map[string]interface{}{}
-
-	// patch envoyfilter according to rules
-	// otherwhise just the original filter will be applied
-	if err == nil && aclExtension.DeletionTimestamp.IsZero() {
-		extSpec := &controller.ExtensionSpec{}
-		if err := json.Unmarshal(aclExtension.Spec.ProviderConfig.Raw, extSpec); err != nil {
-			return admission.Errored(http.StatusInternalServerError, err)
+	// if an error occured or the extension is in deletion, just allow without
+	// introducing any patches
+	if err != nil || !aclExtension.DeletionTimestamp.IsZero() {
```

Comment:
```suggestion
	if err != nil || aclExtension.DeletionTimestamp.IsZero() {
```
> if an error occured or the extension is in deletion

Either the comment is wrong, the code is wrong or it's too early for me.

Reply:
maybe the commend should say

> NOT in deletion

?

Reply:
oh I see. Atleast I provided all the options hehe